methods_and_recursion/bottles_of_beer.rb

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+
+def song(num_bottles)
+  if num_bottles > 2
+    print "#{num_bottles} bottles of beer on the wall, "
```

Comment:
Seems like a lot of duplication here... Maybe you could make a `plural(number, string)` method that would return the singular or plural version of string depending on what number is?